### DISCUSSION TITLE
Vietnamese support

### DISCUSSION THREAD

**Discussion by hckhanh** [oramasearch/orama#651] [Q&A] [answered]
I really want to ask about the roadmap for the upcoming supported languages. Is Vietnamese available?

**Comment by micheleriva** [comment, marked as answer]
Hi!
I think we could probably try to add official support to Vietnamese via https://github.com/vunb/vntk or some other similar toolkit.

This means:

1. Exposing a Vietnamese tokenizer
2. Exposing a Vietnamese stemmer
3. Exposing a list of Vietnamese stop-words

You can find a reference implementation here: https://github.com/askorama/orama/blob/main/packages/tokenizers/src/tokenizer-mandarin/src/tokenizer.ts

It would really help if a native speaker could help with this. We'd try our best to support any contribution in that sense.

**Reply by hckhanh** [reply]
Thank you @micheleriva.
I am a Vietnamese. Perhaps, I think I can help you with this.

**Reply by micheleriva** [reply]
@hckhanh any help is highly appreciated 🙏
